docs/src/pages/components/icons/icons.md

```diff
@@ -40,28 +40,62 @@ The configuration of a Babel plugin is encouraged.
 
 Each icon also has a "theme": `Filled` (default), `Outlined`, `Rounded`, `Two tone` and `Sharp`. If you want to import the icon component with a "theme" different than default, append the "theme" name to the icon name. For example `@material-ui/icons/Delete` icon with:
```

Comment:
Could possibly remove the back-ticks around the theme names (or italicize them instead).

Reply:
I have tried to keep the backtick around `@material-ui/icons/X`. It seems to be acceptable now, as we have removed the others. I feel that the page can accept a small number of "backticked" before it gets overwhelming. What do you think of the output with the latest commit? Should we remove it here too?

Reply:
> I have tried to keep the backtick around @material-ui/icons/X

That doesn't seem to be the case. See my suggestion for what I meant.